src/test/java/seedu/duke/watchlist/WatchlistTest.java

```diff
@@ -2,66 +2,39 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import seedu.duke.bookmark.Bookmark;
 
 import java.util.ArrayList;
 
 import static junit.framework.Assert.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 
 class WatchlistTest {
-    private Watchlist emptyWatchlist;
-    private Watchlist filledWatchlist;
+    private Watchlist firstWatchlist;
+    private Watchlist secondWatchlist;
 
     @BeforeEach
     void setUp() {
         ArrayList<Integer> animeList = new ArrayList<>();
-        animeList.add(0);
+        animeList.add(1);
```

Comment:
Perhaps use of a constant would provide more clarity.

Reply:
Noted on that, I have updated to use a constant for it.